susemanager-utils/susemanager-sls/src/modules/mgractionchains.py

```diff
@@ -138,7 +162,7 @@ def start(actionchain_id):
         raise CommandExecutionError(ret)
     return ret
 
-def next(actionchain_id, chunk, next_action_id=None, ssh_extra_filerefs=None):
+def next(actionchain_id, chunk, next_action_id=None, ssh_extra_filerefs=None, reboot_required=None):
```

Comment:
Maybe set the default value for reboot_required to "False" ?

Reply:
yes, changed :)